model/jpa/pom.xml

```diff
@@ -84,7 +84,42 @@
                     <groupId>com.h2database</groupId>
                     <artifactId>h2</artifactId>
                     <scope>test</scope>
-                </dependency>                
+                </dependency>
+
+                <dependency>
+                    <groupId>org.jboss.spec</groupId>
+                    <artifactId>jboss-javaee-6.0</artifactId>
```

Comment:
> 

we could pick those parts of JAvaEE that we really need. This caused download of a lot of stuff, things we do not use/need:

```
Downloading: http://repository.jboss.org/nexus/content/groups/public/org/jboss/spec/javax/el/jboss-el-api_2.2_spec/1.0.2.Final/jboss-el-api_2.2_spec-1.0.2.Final.pom
Downloaded: http://repository.jboss.org/nexus/content/groups/public/org/jboss/spec/javax/el/jboss-el-api_2.2_spec/1.0.2.Final/jboss-el-api_2.2_spec-1.0.2.Final.pom (5 KB at 9.9 KB/sec)
Downloading: http://repository.jboss.org/nexus/content/groups/public/org/jboss/spec/javax/enterprise/deploy/jboss-jad-api_1.2_spec/1.0.1.Final/jboss-jad-api_1.2_spec-1.0.1.Final.pom
Downloaded: http://repository.jboss.org/nexus/content/groups/public/org/jboss/spec/javax/enterprise/deploy/jboss-jad-api_1.2_spec/1.0.1.Final/jboss-jad-api_1.2_spec-1.0.1.Final.pom (5 KB at 11.4 KB/sec)
Downloading: http://repository.jboss.org/nexus/content/groups/public/org/jboss/spec/javax/xml/registry/jboss-jaxr-api_1.0_spec/1.0.2.Final/jboss-jaxr-api_1.0_spec-1.0.2.Final.pom
Downloaded: http://repository.jboss.org/nexus/content/groups/public/org/jboss/spec/javax/xml/registry/jboss-jaxr-api_1.0_spec/1.0.2.Final/jboss-jaxr-api_1.0_spec-1.0.2.Final.pom (5 KB at 10.7 KB/sec)
Downloading: http://repository.jboss.org/nexus/content/groups/public/org/jboss/spec/javax/xml/soap/jboss-saaj-api_1.3_spec/1.0.2.Final/jboss-saaj-api_1.3_spec-1.0.2.Final.pom
Downloaded: http://repository.jboss.org/nexus/content/groups/public/org/jboss/spec/javax/xml/soap/jboss-saaj-api_1.3_spec/1.0.2.Final/jboss-saaj-api_1.3_spec-1.0.2.Final.pom (5 KB at 10.1 KB/sec)
Downloading: http://repository.jboss.org/nexus/content/groups/public/org/jboss/spec/javax/xml/ws/jboss-jaxws-api_2.2_spec/2.0.1.Final/jboss-jaxws-api_2.2_spec-2.0.1.Final.pom
Downloaded: http://repository.jboss.org/nexus/content/groups/public/org/jboss/spec/javax/xml/ws/jboss-jaxws-api_2.2_spec/2.0.1.Final/jboss-jaxws-api_2.2_spec-2.0.1.Final.pom (4 KB at 7.4 KB/sec)
Downloading: http://repository.jboss.org/nexus/content/groups/public/com/sun/faces/jsf-api/2.0.3-b05/jsf-api-2.0.3-b05.pom
Downloading: https://repository.jboss.org/nexus/content/repositories/snapshots/com/sun/faces/jsf-api/2.0.3-b05/jsf-api-2.0.3-b05.pom
```